### DISCUSSION TITLE
[BUG] Formulario compartido (externo) no se visualiza los survey/survey

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#297] [4. Bug Reports] [answered]
Hola en una implementacion con @AlejandroLoraG colocamos un survey/survey, lo probamos internamente y funciona. Sin embargo al usar enlace externo, no lo toma.

Adjunto link video: https://drive.google.com/file/d/1_XEDoN-gT0IKjl5CWyCbGYZNUjzYW4KS/view?usp=sharing

Gracias =>

**Comment by NicolasPLG** [comment]
Hola alguna novedad de esto? :O?

Gracias =>

**Comment by guillevalin** [comment, marked as answer]
Lamentablemente es una limitación actual de los survey+survey. Se creará un issue.

**Reply by NicolasPLG** [reply]
Muchas gracias @guillevalin
